tasks/FindMyMates.tex

```diff
@@ -21,18 +21,21 @@ \subsection*{Setup}
 
 \subsection*{Procedures}
 \begin{enumerate}
-	\item Robot finds and reports description and location of two guests.
-	\item \textbf{Optionally,} robot reports description and location of the third guest.
+	\item The robot finds and reports the description and location of two guests.
+	\item \textbf{Optionally,} the robot reports the description and location of the third guest.
 \end{enumerate}
 
 \subsection*{Additional Rules and Remarks}
 \begin{enumerate}[nosep]
 	\item \textbf{Reporting:} Points are scored only when the robot goes back to the operator and delivers the report on time.
 
-	\item \textbf{Description of a person:} The description of each guest should be unique, i.e. not shared with any other guest. This may include clothing, age, height, complexion, skin or hair color, gender, etc.
+    \item \textbf{Description of a person:} The description of each guest should be unique, i.e. not shared with any other guest. This may include clothing, age, height, complexion, skin or hair color, gender, etc.
 
 	\item \textbf{Description of a location of a person:} A relative location based on the surroundings (e.g. sitting on the couch, standing beside the lamp, etc.).
 
+    \item \textbf{Description format:} The description of a person and their location should be provided in a natural language form (ideally, spoken out by the robot, or alternatively displayed on the robot's screen).
```

Comment:
This is completely against the mind of the rulebook. The rulebook should not enforce one way of solving a problem.

Reply:
I don't see how this enforces one way of solving the problem, as it simply clarifies the task **in accordance with the rest of the task description**. I particularly don't understand how anything other than a natural language description would fit the requirements of the task (I suppose one could argue that an image satisfies the location requirement, but certainly not the unique description requirement).

In any case, I will leave it up to @myszzka96 to decide what she wants to do about this, as she was looking at this task more closely.

Reply:
"4. Location of a Person: A relative location based of the surroundings (e.g. The kid is
sitting on the coach, The tall man is standing beside the lamp, etc.)." is exactly what was in the 2019 rule book. I am fine with the suggested change by @MatthijsBurgh as to allowing different description formats and adding that note into the rule book.

Reply:
I have removed that part in the latest commit.